Enrich_methods/example.py

```diff
@@ -0,0 +1,210 @@
+import argparse
+from pyne import material
+from pyne.material import Material, MultiMaterial
+from pyne.material_library import MaterialLibrary
+from material_db_tools import mix_by_volume
+import material_db_tools as mdbt
+from material_library import (
+    mat_data as pure,
+)  # this is importing the material dictionaries
+
+mat_lib_test = MaterialLibrary()
+# you may need to adjust this path for your setup
+mat_lib_test.from_json("material_library.json")
+
+
+simulation_lib = MaterialLibrary()
+
+
+################################### Method 3
```

Comment:
Definitely don't like this one, and we've moved away from this basic pattern elsewhere.